plugins/querytranslate/util.go

```diff
@@ -341,6 +345,7 @@ type Settings struct {
 	EnableSearchRelevancy *bool                   `json:"enableSearchRelevancy,omitempty"`
 	UseCache              *bool                   `json:"useCache,omitempty"`
 	QueryRule             *map[string]interface{} `json:"queryRule,omitempty"`
+	Backend               *string                 `json:"backend,omitempty"`
```

Comment:
@deepjyoti30 Use `enum` so it can auto-validate the valid inputs. You can follow the `type` property in Query struct.